content/en/docs/tasks/administer-cluster/kubelet-in-userns.md

```diff
@@ -41,13 +41,12 @@ See [Running kind with Rootless Docker](https://kind.sigs.k8s.io/docs/user/rootl
 
 ### minikube
 
-[minikube](https://minikube.sigs.k8s.io/) also supports running Kubernetes inside Rootless Docker.
+[minikube](https://minikube.sigs.k8s.io/) also supports running Kubernetes inside Rootless Docker or Rootless Podman.
```

Comment:
Hi @AkihiroSuda . Thanks for your contribution.
I think it makes sense to update the Kubernetes docs and link to the `minikube` docs. The `minikube` docs describe how to use the experimental podman driver.
/lgtm